### PR TITLE
[CHORE] Updated cliffy's CDN source & incremented CLI version

### PR DESCRIPTION
**Pull request by t8**

### CODE REVIEW

**Review by SteelAlloy — APPROVED** [review]
LGTM

File: src/version.ts [hunk 1]
```diff
@@ -1 +1 @@
-export const version = "0.1.8"; // TODO(@qu4k): ugly
+export const version = "0.1.9"; // TODO(@qu4k): ugly
```
Comment (notfilippo) [inline src/version.ts:1]: I think we should update version once the rewrite lands.

File: src/version_test.ts [hunk 2]
```diff
@@ -7,6 +7,6 @@ import { version } from "./version.ts";
 Deno.test({
   name: "internal | version | number",
   fn(): void {
-    assertEquals(version, "0.1.8");
+    assertEquals(version, "0.1.9");
```
Comment (notfilippo) [inline src/version_test.ts:10]: ditto

**Review by notfilippo — CHANGES_REQUESTED** [review]

File: src/version.ts (hunk 1 above)
Comment (SteelAlloy) [reply src/version.ts:1]: We'll do another version after the rewrite.
Currently, no version of eggs works under Deno 1.2, we had to release this version as soon as possible.

Comment (notfilippo) [reply src/version.ts:1]: Understandable. Good to go! 
Besides I like 0.2.0 for the rewrite 🌟

**Review by notfilippo — APPROVED** [review]
LGTM. 🚀

**Pull request merged**